src/utils/parser.ts

```diff
@@ -39,3 +40,41 @@ export function nodeToString(node: Node, code: string) {
   // Copy the string value directly from the input code
   return code.slice(start, end).trim();
 }
+
+/**
+ * Finds all css delcarations
+ * @param root PostCss AST Root
+ */
+function getCssProps(root: Root): Declaration[] {
+  const { nodes } = root;
+
+  if (!nodes) {
+    return [];
+  }
+
+  function iterate(nodes: ChildNode[], declarations: Declaration[] = []) {
```

Comment:
Nitpick: I think a better name for this function would be `recurse`, since that's really what you're doing.